MoneySplitter.Services/Mapper.cs

```diff
@@ -43,8 +41,40 @@ public RegisterUserData ConvertRegisterModelToDataRegisterUser( RegisterModel re
                 ImageBase64String = registerModel.ImageBase64String,
                 BackgroundImageBase64String=registerModel.BackgroundImageBase64String
             };
+        }
 
-            return dataRegisterUser;
+        public RegisterTransactionData ConvertRegisterTransactioModelToRegisterTransactionData(RegisterTransactionModel registerTransactionModel)
+        {
+            return new RegisterTransactionData()
+            {
+                Title = registerTransactionModel.Title,
+                Token=registerTransactionModel.Token,
+                Email = registerTransactionModel.Email,
```

Comment:
Please use spaces between =
Title=title - bad
Title = title - good
It is not really important, but looks dirty :)

Reply:
add spaces